app/scripts/load_manager.py

```diff
@@ -36,6 +41,10 @@ def create_task(job_id, cf_handler=None):
     if cf_handler is None:
         cf_handler = create_cf_handler()
 
+    if size != "small":
+        task_contract["memory_in_mb"] = TASK_SIZE_ENUM[size][0]
+        task_contract["disk_in_mb"] = TASK_SIZE_ENUM[size][1]
+
```

Comment:
This will give a key error if size is something other than `medium` or `large`. Suggest validating before utilizing.

Reply:
Would you suggest: `if size != "small and TASK_SIZE_ENUM[size]`?

Reply:
I think I would suggest `if size != "small" and size in TASK_SIZE_ENUM`

Reply:
quick change incoming